parser/fuzz/build.rs

```diff
@@ -0,0 +1,14 @@
+use std::error::Error;
+use std::fs::File;
+use std::path::Path;
+
+fn main() -> Result<(), Box<dyn Error>> {
+	let out_path = Path::new(&std::env::var("OUT_DIR")?).join("common.rs");
+	let mut out = File::create(out_path)?;
+	reqwest::blocking::get(
+		"https://raw.githubusercontent.com/boa-dev/boa/main/fuzz/fuzz_targets/common.rs",
```

Comment:
Does this request on every build? Will this get rate-limited?

Reply:
No, only when the build stage hasn't been run already.